source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -339,6 +383,794 @@ TEST_F(NiRFmxWLANDriverApiTests, TXPFromExample_FetchData_DataLooksReasonable)
+  EXPECT_EQ(2, ofdm_mod_acc_fetch_ppdu_type_response.ppdu_type());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_guard_interval_type_response);
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_lsig_parity_check_status_response);
+  EXPECT_EQ(-1, ofdm_mod_acc_fetch_lsig_parity_check_status_response.lsig_parity_check_status());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_sigcrc_status_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_sigcrc_status_response.sig_crc_status());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_sigbcrc_status_response);
+  EXPECT_EQ(-1, ofdm_mod_acc_fetch_sigbcrc_status_response.sigb_crc_status());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_number_of_users_response);
+  EXPECT_EQ(1, ofdm_mod_acc_fetch_number_of_users_response.number_of_users());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_mcs_index_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_mcs_index_response.mcs_index());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_number_of_space_time_streams_response);
+  for (int i = 0; i < numberOfUsers; i++)
+  {
+    EXPECT_EQ(0, spaceTimeStreamOffsetArray[i]);
+  }
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_mcs_index_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_mcs_index_response.mcs_index());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_number_of_space_time_streams_response);
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_frequency_error_mean_response);
+  EXPECT_GT(0.0, ofdm_mod_acc_fetch_frequency_error_mean_response.frequency_error_mean());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_symbol_clock_error_mean_response);
+  EXPECT_GT(0.0, ofdm_mod_acc_fetch_symbol_clock_error_mean_response.symbol_clock_error_mean());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_stream_rmsevm_response);
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_response.stream_rms_evm_mean());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_response.stream_data_rms_evm_mean());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_response.stream_pilot_rms_evm_mean());
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response);
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.x0());
+  EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.dx());
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.stream_rms_evm_per_subcarrier_mean_size());
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.stream_rms_evm_per_subcarrier_mean().size());
+  // EXPECT_EQ(0.0, ofdm_mod_acc_fetch_stream_rmsevm_per_subcarrier_mean_trace_response.stream_rms_evm_per_subcarrier_mean(0));
+  EXPECT_SUCCESS(session, ofdm_mod_acc_fetch_pilot_constellation_trace_response);
+  EXPECT_EQ(0, ofdm_mod_acc_fetch_pilot_constellation_trace_response.pilot_constellation().size());
+  // EXPECT_EQ(0.0, ofdm_mod_acc_fetch_pilot_constellation_trace_response.pilot_constellation(0).real());
```

Comment:
im assuming these were not zero?

Reply:
The array size is 0 (see the line above), so this query for the first array element would fail if uncommented. I did the same (left in a commented line) in the NR tests. Either we'll figure out how the setup is wrong and fix it, in which case this commented line will be a reminder to test the value, or else we'll never fix the setup, and the commented line will just be pointing out that there's no data values to test. That's _my_ opinion on the matter, anyway.

Reply:
Yeah, it's because the array sizes are 0. David made the point in his NR tests PR that he was just going to leave them around and I liked that.

Like David said, either
1. At some point we have array sizes and are reminded to assert on at least on of the array values.
2. It remains forever and is a minor visual reminder that we got an empty array for something.